cockpit/src/subscriptions-client.js

```diff
@@ -44,6 +51,16 @@ client.config = {
     loaded: false,
 };
 
+client.syspurposeStatus = {
+    info : {
+        "service_level_agreement" : null,
+        "usage_type" : null,
+        "role" : null,
+        "add-ons" : null,
```

Comment:
Non-nacking comment: I'd prefer this were "addons" to be in line with the subscription-manager tooling.